vizro-core/docs/pages/user_guides/tabs.md

```diff
@@ -0,0 +1,128 @@
+# How to use tabs
+
+Tabs organize and separate groups of related content in a dashboard, letting users navigate between different sections or views within a click.
+This guide shows you how to use tabs to organize your containers into subsections inside the dashboard.
+
+To add [`Tabs`][vizro.models.Tabs] to your page, do the following:
+
+1. Insert the [`Tabs`][vizro.models.Tabs] into the `components` argument of the [`Page`][vizro.models.Page]
+2. Insert your [`Containers`][vizro.models.Container] into the `components` argument of the [`Tabs`][vizro.models.Tabs]
+
+By using the [`Tabs`][vizro.models.Tabs], the following applies:
+- [`Filters`][vizro.models.Filter] affect all components on all tabs (opened and closed) of the page if not specified otherwise inside `Filter.targets`
+- The `title` of the outer [`Container`][vizro.models.Container] inserted into `Tabs.components` will be displayed as a tab label, and the title will removed from the `Container`
+- If you want to keep the `Container.title` inside the Tab, you can apply custom CSS
+- If you want a different `Container.title` inside the tab, add a nested [`Container`][vizro.models.Container]
```

Comment:
I would actually even just remove these two points because "you can apply custom CSS" isn't super helpful if we don't specify what that CSS is, which means adding more detail than most people will care about, and the nested `Container` point still sounds like an unusual requirement to me and isn't fully explained here either. I would rather wait and see who (if anyone) asks about these two points. And if they do add a "proper" section on them.

If we really want them here already though then can we break the list up into two separate things instead of having the `Filter` point in the same list as the `title` ones? e.g. put the `Filter` thing in a separate note and then just have this list consist of the 3 points about title.

Reply:
No, I agree - let's just remove the two points and wait if people do request it 👍